modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -217,7 +217,7 @@ public void testGetObjectEntry() throws Exception {
 		String title = RandomTestUtil.randomString();
 
 		ObjectEntry objectEntry = _objectEntryManager.addObjectEntry(
-			_getDTOConverterContext(), _objectDefinition,
+			dtoConverterContext, _objectDefinition,
```

Comment:
I think you can call `_getDTOConverterContext();` directly, it is used only twice inside this method.